src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let memory = [];
```

Comment:
Storing all numbers in an array might not be the most memory-efficient way to handle large sequences of additions. Instead, consider keeping a running total.